subprojects/docs/build.gradle

```diff
@@ -16,6 +17,11 @@ plugins {
     id 'gradlebuild.split-docs'
 }
 
+class DocsBuildFlags {
```

Comment:
@blindpirate would you prefer having these flags here or in `gradlebuild.basics.BuildParams`?

Reply:
Oh, yes, please put everything in `BuildParams`.